backend/organization/views/project_views.py

```diff
@@ -173,9 +175,31 @@ def post(self, request):
         except ProjectStatus.DoesNotExist:
             return Response({
                 'message': "Passed status {} does not exist".format(request.data["status"])
-            })
+            })  
 
-        project = create_new_project(request.data)
+        try:
+            translations = get_project_translations(request)            
+        except ValueError:
+            return Response({
+                'message': "Please use the same language for all texts."
+            }, status=status.HTTP_400_BAD_REQUEST)  
```

Comment:
I wouldn't return 400 error here if translation fails here. But log the error maybe somehow notify us to see what went wrong. The goal here is to allow users to create projects. Translation should not be a blocker on creating projects.

Reply:
Good point!